tools/workflow/docker-version-compare.sh

```diff
@@ -0,0 +1,95 @@
+#! /bin/bash
```

Comment:
Should we write this in go?

Reply:
Thought about it, but it seemed simpler this way.